File: app/src/main/java/com/overswayit/plesnisavezsrbije/activities/AdjudicatorsActivity.java
package com.overswayit.plesnisavezsrbije.activities;

import android.content.res.ColorStateList;
import android.os.Bundle;

import androidx.appcompat.widget.Toolbar;
import androidx.viewpager.widget.ViewPager;

import com.google.android.material.tabs.TabLayout;
import com.overswayit.plesnisavezsrbije.R;
import com.overswayit.plesnisavezsrbije.fragments.AdjudicatorsAdapter;

import butterknife.BindView;
import butterknife.ButterKnife;

public class AdjudicatorsActivity extends BaseActivity {

    @BindView(R.id.tab_layout)
    TabLayout tabLayout;

    @BindView(R.id.pager)
    ViewPager viewPager;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_adjudicators);
        ButterKnife.bind(this);

        Toolbar toolbar = findViewById(R.id.toolbar);
        setSupportActionBar(toolbar);
        toolbar.setNavigationIcon(R.drawable.ic_arrow_back);
        toolbar.setNavigationOnClickListener(v -> onBackPressed());
        toolbar.setTitle(R.string.adjudicators);

        tabLayout.addTab(tabLayout.newTab().setText("Latino i Standard"));
        tabLayout.addTab(tabLayout.newTab().setText("Savremeni"));
        tabLayout.setTabGravity(TabLayout.GRAVITY_FILL);
        tabLayout.setTabTextColors(getResources().getColor(R.color.white90), getResources().getColor(R.color.white));

        final AdjudicatorsAdapter adapter = new AdjudicatorsAdapter
                (getSupportFragmentManager(), tabLayout.getTabCount());
        viewPager.setAdapter(adapter);
        viewPager.addOnPageChangeListener(new TabLayout.TabLayoutOnPageChangeListener(tabLayout));
        tabLayout.setOnTabSelectedListener(new TabLayout.OnTabSelectedListener() {
            @Override
            public void onTabSelected(TabLayout.Tab tab) {
                viewPager.setCurrentItem(tab.getPosition());
            }

            @Override
            public void onTabUnselected(TabLayout.Tab tab) {

            }

            @Override
            public void onTabReselected(TabLayout.Tab tab) {

            }
        });
    }
}

File: app/src/main/java/com/overswayit/plesnisavezsrbije/activities/AdjudicatorsActivity.kt
package com.overswayit.plesnisavezsrbije.activities

import android.os.Bundle
import androidx.databinding.DataBindingUtil.setContentView
import com.google.android.material.tabs.TabLayout
import com.overswayit.plesnisavezsrbije.R
import com.overswayit.plesnisavezsrbije.databinding.ActivityAdjudicatorsBinding
import com.overswayit.plesnisavezsrbije.fragments.AdjudicatorsAdapter
import kotlinx.android.synthetic.main.activity_adjudicators.*

class AdjudicatorsActivity : BaseActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
//        setContentView(R.layout.activity_adjudicators)
        val binding: ActivityAdjudicatorsBinding = setContentView(this, R.layout.activity_adjudicators)

        setSupportActionBar(binding.toolbar)
        toolbar.setNavigationIcon(R.drawable.ic_arrow_back)
        toolbar.setNavigationOnClickListener { onBackPressed() }
        toolbar.setTitle(R.string.adjudicators)

        binding.tabLayout.addTab(tabLayout!!.newTab().setText(getString(R.string.latin_and_standard)))
        tabLayout.addTab(tabLayout!!.newTab().setText(getString(R.string.modern)))
        tabLayout.tabGravity = TabLayout.GRAVITY_FILL
        tabLayout.setTabTextColors(resources.getColor(R.color.white90, null), resources.getColor(R.color.white, null))

        val adapter = AdjudicatorsAdapter(supportFragmentManager, tabLayout!!.tabCount)
        binding.viewPager.adapter = adapter
        viewPager!!.addOnPageChangeListener(TabLayout.TabLayoutOnPageChangeListener(tabLayout))
        tabLayout!!.addOnTabSelectedListener(object : TabLayout.OnTabSelectedListener {
            override fun onTabSelected(tab: TabLayout.Tab?) {
                if (tab != null) {
                    viewPager!!.currentItem = tab.position
                }
            }

            override fun onTabReselected(tab: TabLayout.Tab?) {
                //Ignored
            }

            override fun onTabUnselected(tab: TabLayout.Tab?) {
                //Ignored
            }
        })
    }
}
